soccer/SimFieldView.cpp

```diff
@@ -68,9 +69,17 @@ void SimFieldView::mousePressEvent(QMouseEvent* me) {
 }
 
 void SimFieldView::mouseMoveEvent(QMouseEvent* me) {
+    _posLabel->move(QPoint(me->pos().x() - 45, me->pos().y() + 17));
```

Comment:
It would probably be best to move this code into `FieldView` now that you've moved the other label-related things there.